_source/_posts/2025-01-28-integrator-office-hours.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: blog_post
+title: "Announcing Integrator Office Hours"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "A new opportunity for personalized assistance creating OIN Integrations"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/introduction-to-okta-integration-network/building-integrations.jpg
+type: awareness
+---
+
+Are you working on an integration for the [Okta Integration Network](https://www.okta.com/integrations/)? 
+
+Would you like assistance designing, implementing, or submitting your integration?
+
+Our Integrator Office Hours program is now open to all builders at all stages of integration development! To attend, simply sign up [on Calendly](https://calendly.com/joyce-morrell/okta-integrations-office-hours). If you share more details about your integration when you sign up, we'll make sure to have the right experts at your session! 
+
+Office hours are currently on Tuesdays at 10 am Pacific. You can also ask questions on the [Developer Forum](https://devforum.okta.com/c/questions/6). 
```

Comment:
You might want to add "Pacific time zone" for clarity (making it locale-friendly).